### PR TITLE
ci: test helm chart

### PR DESCRIPTION
**Pull request by paddatrapper**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
```
Comment (yashmehrotra) [inline .github/workflows/test.yml:11]: (nit) ubuntu-latest

File: chart/templates/kratos-config.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kratos-config
+  name: mission-control-kratos-config
```
Comment (yashmehrotra) [inline chart/templates/kratos-config.yaml:7]: Is there a reason for this ?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          helm dependency build ./chart
+          helm repo add flanksource https://flanksource.github.io/charts
+          helm repo add ory https://k8s.ory.sh/helm/charts
+          ct install --charts ./chart --namespace default
+
+      - name: Test deploying topology
+        run: |
+          kubectl apply -f topologies/mission-control.yaml
+          kubectl get topology mission-control -o yaml
+          kubectl wait --for=condition=ready --timeout=5m topology mission-control
```
Comment (yashmehrotra) [inline .github/workflows/test.yml:69]: Should we also execute a shell test via curl, like for testing some data in incidents or peoples table etc. ?

File: chart/templates/kratos-config.yaml (hunk 2 above)
Comment (paddatrapper) [reply chart/templates/kratos-config.yaml:7]: Yes - we cannot overwrite the configmap created by the Kratos chart (see https://github.com/flanksource/mission-control-chart/actions/runs/4859934069/jobs/8663167611#step:7:39 for example). So this creates our config in a new configmap and maps that to a second config file that takes preference over the one provided by the Kratos chart

File: .github/workflows/test.yml (hunk 3 above)
Comment (paddatrapper) [reply .github/workflows/test.yml:69]: Sure. If you suggest an endpoint to test I can add that

**Review by yashmehrotra — APPROVED** [review]

**Pull request merged**